hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestHddsVolumeChecker.java

```diff
@@ -91,6 +92,7 @@
 
   @Before
   public void setup() throws IOException {
+    DefaultMetricsSystem.instance().shutdown();
```

Comment:
Do we still need this one after the added cleanup in TestHddsDispatcher?

Reply:
My mistake, I will remove this